e2e/stateful-pallet-storage/handlePaginated.test.ts

```diff
@@ -232,22 +232,6 @@ describe('📗 Stateful Pallet Storage', function () {
         section: 'statefulStorage',
       });
     });
-
-    it('🛑 should fail call to remove page for un-delegated attempts', async function () {
```

Comment:
These tests are not correct and I had to remove since we are using a `badMsaId` which it's nonce is lower than current nonce for providerKeys it is throwing following error. The only reason this was not happening before was because it was erroring out on the stalehash check before getting to this error.

`1010: Invalid Transaction: Transaction is outdated` which might be a nonce check race condition issue which is not related to this specific pallet or extrinsic

Reply:
On a second thought it is weird why this gets triggered

The following is the only place we are returning `InvalidTransaction::Stale`.
```
		// The default account (no account) has a nonce of 0.
		// If account nonce is not equal to the tx nonce (self.0), the tx is invalid.  Therefore, check if it is a stale or future tx.
		if self.0 != account.nonce {
			return Err(if self.0 < account.nonce {
				InvalidTransaction::Stale
			} else {
				InvalidTransaction::Future
			}
			.into())
		}
```

Reply:
I don't think it's a blocker for this PR merge but we should take a deeper look